rock.osdeps

```diff
@@ -34,7 +34,10 @@ binding_of_caller:
     gem: binding_of_caller
 
 activesupport:
-    gem: activesupport <= 4.2.5.1
+  ubuntu: ruby-activesupport
+  debian: ruby-activesupport
+  default:
+    gem: activesupport
```

Comment:
Then the right fix is to have a per-ruby-version definition of activesupport (look at the rock.osdeps-ruby* files). Many people don't use the prepackaged version of Ruby.